### PR TITLE
added author strifale to osci_2023

### PR DESCRIPTION
**Pull request by strifale**
### Description
Add new Author strifale
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/strifale.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Sharvil Trifale
+short_name: Sharvil
+github: strifale
+linkedin: linkedin.com/in/strifale
```
Comment (cwperks) [inline _authors/osci_2023/strifale.markdown:5]: Hi @strifale , please only include the linkedin username and not a full URL

**Review by cwperks — CHANGES_REQUESTED** [review]

File: _authors/osci_2023/strifale.markdown (hunk 1 above)
Comment (cwperks) [reply _authors/osci_2023/strifale.markdown:5]: Otherwise this LGTM!

Comment (strifale) [reply _authors/osci_2023/strifale.markdown]: @cwperks  updated!

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Pull request merged**